### ISSUE TITLE
[Controller Request]: Steelseries Nimbus+

### ISSUE THREAD

**Issue by 0skolok** [ndeadly/MissionControl#786]
### Controller Name

Steelseries Nimbus+

### Current Controller Behaviour

Controller successfully pairs with the console but doesn't respond to inputs

### Supported Features

- [X] Rumble/Vibration
- [X] Motion Controls
- [X] Player Number Indicator
- [X] Battery Level Reporting

### Additional Details

https://steelseries.com/gaming-controllers/nimbus-plus

**Comment by ndeadly** [comment]
This might just need to have a new vendor/product id whitelisted. Can you update your post to include these values? Since the controller paired, you can grab them with my [btdb](https://github.com/ndeadly/MissionControl/files/7227157/btdb-1300.zip) tool

**Comment by 0skolok** [comment]
@ndeadly 

Error Code: 2021-0003 (0x615)
Program: 0104444444444000
Firmware: 17.0.1 (Atmosphère 1.6.2-master-f7bf379cf)
An error has occurred.
Please press the POWER Button to restart the console normally, or a VOL button
restate te console no the POWER Butson est 12 seconds to tum the console off.
If the problem persists, refer to the Nintendo Support Website.
support.nintendo.com/switch/error
General Purpose Registers
...

**Comment by ndeadly** [comment]
Why are you sharing this? That title id has nothing to do with Mission Control, and without context sharing an error report for a controller request makes no sense.

**Comment by 0skolok** [comment]
@ndeadly Hi. Sorry, I didn't write details... When I try to run the btdb tool I get the fatal error.
I found the error report log. Or do you need a dump?

> Atmosphère Fatal Report (v1.1):
> Result:                          0x615 (2021-0003)
> 
> Program ID:                      0104444444444000
> Process Name:                    Homebrew Me
> Firmware:                        17.0.1 (Atmosphère 1.6.2-master-f7bf379cf)
> General Purpose Registers:
> Start Address:                   0000000000000000
> Stack Trace:
>         ReturnAddress[00]:       0000000000000000
>         ReturnAddress[01]:       0000000000000000
>         ReturnAddress[02]:       0000000000000000
>         ReturnAddress[03]:       0000000000000000
>         ReturnAddress[04]:       0000000000000000
>         ReturnAddress[05]:       0000000000000000
>         ReturnAddress[06]:       0000000000000000
>         ReturnAddress[07]:       0000000000000000
>         ReturnAddress[08]:       0000000000000000
>         ReturnAddress[09]:       0000000000000000
>         ReturnAddress[10]:       0000000000000000
>         ReturnAddress[11]:       0000000000000000
>         ReturnAddress[12]:       0000000000000000
>         ReturnAddress[13]:       0000000000000000
>         ReturnAddress[14]:       0000000000000000
>         ReturnAddress[15]:       0000000000000000
>         ReturnAddress[16]:       0000000000000000
>         ReturnAddress[17]:       0000000000000000
>         ReturnAddress[18]:       0000000000000000
>         ReturnAddress[19]:       0000000000000000
>         ReturnAddress[20]:       0000000000000000
>         ReturnAddress[21]:       0000000000000000
>         ReturnAddress[22]:       0000000000000000
>         ReturnAddress[23]:       0000000000000000
>         ReturnAddress[24]:       0000000000000000
>         ReturnAddress[25]:       0000000000000000
>         ReturnAddress[26]:       0000000000000000
>         ReturnAddress[27]:       0000000000000000
>         ReturnAddress[28]:       0000000000000000
>         ReturnAddress[29]:       0000000000000000
>         ReturnAddress[30]:       0000000000000000
>         ReturnAddress[31]:       0000000000000000

**Comment by ndeadly** [comment]
Error 0x615 is `sm: too many sessions`. Presumably this is because of the reduced number of `btm` sessions from firmware 17.0.0 onwards. I wasn't able to reproduce on latest firmware, but try disabling Mission Control and/or other homebrew sysmodules before running btdb and see whether you still get the error.

**Comment by 0skolok** [comment]
@ndeadly btdb starts when I disable Mission Control

**Comment by ndeadly** [comment]
Yeah, thought that would be the case. I guess I will have to move the functionality used by btdb into the mission control sysmodule and access it via that instead.

Now that it's working I need you to share the vendor and product ID displayed in btdb. It's the number in the format `XXXX:XXXX`. Or you can just upload a screenshot.
